templates/welcome_email.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+</head>
+
+<body>
+    <h3>Hi {{firstName}},</h3>
```

Comment:
Let's not wrap this in a header element; `<p>` should probably be fine, but will want a space between it and the paragraph starting below it.